src/components/ErrorNotification.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect } from 'react';
+import cn from 'classnames';
+
+type Props = {
+  errorMsg: string;
+  onErrorDelete?: () => void;
+};
+
+export const ErrorNotification: React.FC<Props> = React.memo(({
+  errorMsg,
+  onErrorDelete = () => { },
+}) => {
+  useEffect(() => {
+    const timeoutID = setTimeout(onErrorDelete, 3000);
+
+    return () => {
+      clearTimeout(timeoutID);
+    };
+  }, []);
```

Comment:
fix eslint warning wiht missing dependency
![Screenshot from 2023-11-26 16-21-39](https://github.com/mate-academy/react_todo-app-loading-todos/assets/33029864/cb15da61-3d82-4524-92fb-600e990d6d48)

Reply:
but I dont need side effects after rendering. start the timer only after it appears and stop if the user closed the message earlier. without dependency array the timers will be start after every render